setup.py

```diff
@@ -15,4 +15,18 @@
     packages=find_packages("src"),
     package_dir={"": "src"},
     install_requires=[],
+    zip_safe=False,
+    include_package_data=True,
+    classifiers=[
+        "Development Status :: 1 - Planning",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: Implementation",
+        "Programming Language :: Python :: Implementation :: PyPy",
+        "Topic :: Scientific/Engineering",
```

Comment:
I think this is a relevant `Topic` for the SDK, but I don't know if it applies in general to stdlib-utils